lib/pageflow/file_types.rb

```diff
@@ -7,13 +7,16 @@ def initialize(page_types)
     end
 
     def each(&block)
-      @page_types.map(&:file_types).flatten.uniq(&:model).each(&block)
+      first_level_file_types = @page_types.map(&:file_types).flatten
+      lower_level_file_types = search_for_nested_file_types(first_level_file_types)
+      (first_level_file_types + lower_level_file_types).uniq(&:model).each(&block)
     end
```

Comment:
So this is only tested indirectly, by making sure nested file types show up in the config seed, right? Might be worth a test of its own.